src/lib/parse/CompoundValueNode.cpp

```diff
@@ -742,7 +741,10 @@ std::string CompoundValueNode::toJson(bool toplevel)
 
   if (parentIsObject)
   {
-    return toJsonString(name) + ":" + out;
+    std::string preOut = "\"";
+    preOut += toJsonString(name);
+    preOut += "\":";
+    return preOut + out;
```

Comment:
Or:
```
preOut += out:
return preOut;
```
If you really want to avoid ALL std::string operator+

Reply:
Good catch!

Reply:
Fixed in edd01ffba0453591e991b12d5f62687890108b09